projects/schematics/src/migrations/5_0/constructor-deprecations/data/import-entries-dialog.component.migration.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  ELEMENT_REF,
+  IMPORT_ENTRIES_DIALOG_COMPONENT,
+} from '../../../../shared/constants';
+import { SPARTACUS_CORE } from '../../../../shared/libs-constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const IMPORT_ENTRIES_DIALOG_COMPONENT_MIGRATION: ConstructorDeprecation =
+  {
+    //projects/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts
+    class: IMPORT_ENTRIES_DIALOG_COMPONENT,
+    importPath: SPARTACUS_CORE,
+    deprecatedParams: [],
```

Comment:
There should be a list of all params existing previously 
```suggestion
    deprecatedParams: [
      {
        className: LAUNCH_DIALOG_SERVICE,
        importPath: SPARTACUS_STOREFRONTLIB,
      },
    ],
```

Reply:
I deprecated LAUNCH_DIALOG_SERVICE and added it back after my new ELEMENT_REF parameter which is first.  Correct?

Reply:
You added `LAUNCH_DIALOG_SERVICE` to `addParams` and it's wrong 
Please see suggestions:
https://github.com/SAP/spartacus/pull/15390#discussion_r830300275
https://github.com/SAP/spartacus/pull/15390#discussion_r830300275

Reply:
Thanks.